nservicebus/azure-service-bus/keep-connectionstring-safe.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Keep connection string safe
+summary: Configuring Azure ServiceBus transport to secure connection strings. 
+component: ASB
+tags:
+- Cloud
+- Azure
+- Transports 
+- Security
+reviewed: 2016-04-26
+---
+
+include: keep-connection-string-safe
+
+To enable this behavior, apply the following configuration:
```

Comment:
More info:

As a reader, I can be either a newbie, or someone who's already using nsb.asb and wants to read about this new features.

- For a new user there's no existing context (the fact that connection strings where not safe).
- For an existing user they know what's happening, but they don't know about how safe connection strings work 
  - link to the upgrade guide
- For both type of users, how to configure safe connections

Would this flow make sense?